File: test/core/customer/domain/CustomerIdMother.ts
import {CustomerId} from "../../../../src/core/customer/domain/CustomerId";
import {UuidValueObjectMother} from "../../shared/domain/UuidValueObjectMother";

export class CustomerIdMother {

    static random(): CustomerId {
        return new CustomerId(
            UuidValueObjectMother.random()
        );
    }
}

File: python_django/core_test/customer/domain/customer_id_mother.py
import uuid

from core.customer.domain.customer_id import CustomerId
from core_test.shared.domain.uuid_mother import UUIDMother


class CustomerIdMother:

    @staticmethod
    def random() -> CustomerId:
        random_uuid = UUIDMother.random().value
        return CustomerId(random_uuid)
